integrations/micrometer/micrometer/src/main/java/module-info.java

```diff
@@ -14,10 +14,22 @@
  * limitations under the License.
  */
 
+import io.helidon.common.features.api.Aot;
+import io.helidon.common.features.api.Experimental;
+import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.HelidonFlavor;
+
 /**
  * Support for Micrometer in Helidon SE.
  */
+@Experimental
+@Feature(value = "Micrometer",
+        description = "Micrometer integration",
+        in = HelidonFlavor.SE,
+        path = "Micrometer")
+@Aot
```

Comment:
No need to define `@Aot`, it is supported by default.
Also Micrometer is `@Incubating`, not `@Experimental`

Reply:
Fixed